packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -2087,6 +2107,8 @@ export interface CommonCallingHandlers {
     onCreateLocalStreamView: (options?: VideoStreamOptions) => Promise<void | CreateVideoStreamViewResult>;
     // (undocumented)
     onCreateRemoteStreamView: (userId: string, options?: VideoStreamOptions) => Promise<void | CreateVideoStreamViewResult>;
+    // @beta
```

Comment:
`@public`

Reply:
Won't comment on each one of these, but all `// @beta`'s need updated to result in `@public`